lib/appmap/trace.rb

```diff
@@ -15,34 +15,38 @@ def initialize(package, defined_class, method, static)
 
     class Tracing
       def initialize
-        @tracing = []
+        @tracers = []
       end
 
       def empty?
-        @tracing.empty?
+        @tracers.empty?
       end
 
       def trace(enable: true)
         Tracer.new.tap do |tracer|
-          @tracing << tracer
+          @tracers << tracer
           tracer.enable if enable
         end
       end
 
       def enabled?
-        @tracing.any?(&:enabled?)
+        @tracers.any?(&:enabled?)
+      end
+
+      def last_package_for_current_thread
+        @tracers.map(&:last_package_for_current_thread).first
```

Comment:
Is it the same as
```suggestion
        @tracers.first&.last_package_for_current_thread
```
but slower? ;)